nixos/modules/image/repart-image.nix

```diff
@@ -41,6 +41,25 @@
 }:
 
 let
+  systemdArch = let
+    inherit (stdenvNoCC) hostPlatform;
+  in
+    if hostPlatform.isAarch32 then "arm"
```

Comment:
Is this format documented anywhere? I'd expect this to have lived in `lib` already in some shape or form.

Reply:
repart uses this table to identify architectures: https://github.com/systemd/systemd/blob/main/src/basic/architecture.c#L142

These identifiers are slightly different than other formats we have in lib (e.g. linuxArch, qemuArch, ubootArch). The most notable difference is `x86_64` vs `x86-64`.

I'd be happy to move this to `lib/systems/default` and that's a refactor we can easily do.